saltproc/depcode.py

```diff
@@ -34,12 +35,12 @@ class Depcode(ABC):
     exec_path : str
         Path to depletion code executable.
     template_fname : str
-        Path to user input file for depletion code.
+        Path to template input file for depletion code.
     input_fname : str
-        Name of input file for depletion code rerunning.
+        Path to input file for depletion code rerunning.
```

Comment:
If this variable is actually the path, then it would make more sense for the variable name to include "path" instead of "fname". Same for the "template_fname" variable in line 37.

Reply:
I'll take a look at this.

Reply:
Based on what's in `test_depcode.py`, your description of the behavior is correct. I'll update this.